kore/src/Stats.hs

```diff
@@ -82,3 +87,11 @@ readStats filePath =
     either errorWith return =<< Aeson.eitherDecodeFileStrict filePath
   where
     errorWith message = error ("readStats: " ++ message)
+
+warnIfLowProductivity :: MonadLog log => MonadIO log => log ()
+warnIfLowProductivity = do
+    Stats { gc_cpu_ns, cpu_ns } <- liftIO getStats
+    when (gc_cpu_ns * 10 > cpu_ns) $
+        logWarning
```

Comment:
Is there any reason we don't want to add a separate entry type, `WarnIfLowProductivity`, instead of logging this as a `LogMessage`?

Reply:
parsimony :))
I'll add a new entry type :+1: